go/libcmdline/cmdline.go

```diff
@@ -380,6 +380,16 @@ func (p CommandLine) GetTorProxy() string {
 	return p.GetGString("tor-proxy")
 }
 
+func (p CommandLine) GetProxyType() string {
+	return p.GetGString("proxy-type")
+}
+
+func (p CommandLine) IsSSLPinningEnabled() bool {
```

Comment:
Maybe "TLS" rather than "SSL"?  Or maybe "Cert Pinning"?

Reply:
Changed to cert pinning